cudamapper/src/overlapper.cpp

```diff
@@ -32,8 +33,8 @@ bool overlaps_mergable(const claraparabricks::genomeworks::cudamapper::Overlap o
         return false;
     }
 
-    int query_gap = (o2.query_start_position_in_read_ - o1.query_end_position_in_read_);
-    int target_gap;
+    std::uint32_t query_gap = (o2.query_start_position_in_read_ - o1.query_end_position_in_read_);
```

Comment:
I actually prefer `int` here. `o2.query_start_position_in_read_ - o1.query_end_position_in_read_` should always be a positive value but it's better to not depend on that

Reply:
Yep, this was a bug. Fixed in https://github.com/clara-parabricks/GenomeWorks/pull/453/commits/f994651d9305d441ed6f64f3fbb0949be637d167.